README.md

```diff
@@ -268,6 +268,7 @@ cargo run --bin texture
 ### Utility libraries
 * [vk-sync](https://github.com/gwihlidal/vk-sync-rs) - Simplified Vulkan synchronization logic, written in rust.
 * [vk-mem-rs](https://github.com/gwihlidal/vk-mem-rs) - This crate provides an FFI layer and idiomatic rust wrappers for the excellent AMD Vulkan Memory Allocator (VMA) C/C++ library.
+* [vk-mem-alloc-rs](https://github.com/projectkml/vk-mem-alloc-rs) - A very lightweight wrapper around the AMD Vulkan Memory Allocator
```

Comment:
I'm not sure how this is relevant to the PR.

Reply:
I made another PR with this and I accidently selected the wrong branch, but I removed it so its not added!